lnbits/static/js/components/extension-settings.js

```diff
@@ -0,0 +1,84 @@
+Vue.component('lnbits-extension-settings-form', {
+  name: 'lnbits-extension-settings-form',
+  props: ['options', 'name', 'adminkey', 'endpoint'],
```

Comment:
- `name` is no longer required

```suggestion
  props: ['options', 'adminkey', 'endpoint'],
```